src/guides/v2.3/extension-dev-guide/plugins.md

```diff
@@ -364,11 +364,31 @@ Your plugin classes has this methods:
 | **around**    |                  | aroundDispatch() |                  |
 | **after**     | afterDispatch()  | afterDispatch()  | afterDispatch()  |
 
+`PluginB`::`aroundDispatch()` defines the ($next)[{{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Interception/Interceptor.php] argument with a `callable` type. For example:
+
+```php
+class PluginB
+{
+    public function aroundDispatch(\Magento\Framework\App\Action\Action $subject, callable $next, ...$args)
+    {
+        // The first half of code goes here
+        // ...
+
+        $result = $next(...$args);
+
+        // The second half of code goes here
+        // ...
+
+        return $result;
+    }
+}
+```
+
 The execution will be in this order:
 
 *  `PluginA::beforeDispatch()`
 *  `PluginB::beforeDispatch()`
-*  `PluginB::aroundDispatch()` (Magento calls the first half until `callable`)
+*  `PluginB::aroundDispatch()` (Magento calls the first half after `callable`)
```

Comment:
I'm not sure this statement is 100% correct. The around plugin is supposed to be executed before callable and after callable. In the modified example it's being executed two times before callable. 

```
*  `PluginB::aroundDispatch()` (Magento calls the first half after `callable`)
*  `PluginB::aroundDispatch()` (Magento calls the second half after `callable`)
```

Reply:
Good catch, @rogyar !

**fixed**

Thank you!